test/config.example.js

```diff
@@ -0,0 +1,4 @@
+module.exports = function(reach) {
```

Comment:
this is the file that got deleted that every test calls to set up the reach auth